config/development.json

```diff
@@ -1,15 +1,16 @@
-{
-  "HumanStandardToken": {
-    "total": "1000000000000000000",
-    "name": "Marketing Analytics Token",
-    "symbol": "MAT",
-    "decimals": 9,
-    "price": 49000
-  },
-  "VerifierRegistry": {
-    "tokenAddress": "0x1588d300a9995934aa9daae19be285a66eb46c1c"
-  },
-  "Chain": {
-    "blocksPerPhase": "8"
-  }
-}
+{
+  "HumanStandardToken": {
+    "total": "1000000000000000000",
+    "name": "Marketing Analytics Token",
+    "symbol": "MAT",
+    "decimals": 9,
+    "price": 49000
+  },
+  "VerifierRegistry": {
+    "tokenAddress": "0x1588d300a9995934aa9daae19be285a66eb46c1c"
+  },
+  "Chain": {
+    "blocksPerPhase": "8",
+    "requirePercentOfTokens": "70"
```

Comment:
maybe we make this a floating number and validate that it can only be between 0 and 1

Reply:
I think percent is not a float number, for me is more intuitive to set % as int... but if you really want that I can change :)